### PR TITLE
Username choice: added tests

### PR DESCRIPTION
**Pull request by giommok**

### CODE REVIEW

**Review by magangloff — APPROVED** [review]
All good

File: app/src/main/java/com/github/bgabriel998/softwaredevproject/ProfileActivity.java [hunk 1]
```diff
@@ -94,8 +94,8 @@ public void submitUsernameButton(View view) {
         Log.d("CURRENT_USERNAME", "onSubmit: " + currentUsername);
         if(username.equals(currentUsername)) {
             Log.d("CURRENT_USERNAME", "onSubmit: EQUAL_CASE");
-            Toast toast = Toast.makeText(getApplicationContext(), "You can't choose the username you already have!", Toast.LENGTH_SHORT);
-            toast.show();
+            Snackbar snackbar = Snackbar.make(findViewById(android.R.id.content), "You can't choose the username you already have!" , Snackbar.LENGTH_LONG);
```
Comment (aolsson711) [inline app/src/main/java/com/github/bgabriel998/softwaredevproject/ProfileActivity.java:97]: Should the message be a constant instead of a magic string?

**Review by aolsson711 — COMMENTED** [review]
Small comment regarding a constant. Discuss on Discord. Otherwise everything looks good

File: app/src/main/java/com/github/bgabriel998/softwaredevproject/ProfileActivity.java (hunk 1 above)
Comment (giommok) [reply app/src/main/java/com/github/bgabriel998/softwaredevproject/ProfileActivity.java:97]: Good point. Next week all these magic strings will be on the string.xml file!

**Review by aolsson711 — APPROVED** [review]
Looks good

**Pull request merged**